.travis.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+function build_container(){
+    docker build -t travis/image-inspector-base .
+    cat > Dockerfile.travis <<EOF
```

Comment:
@ilackarms I think we mentioned to split this to another file. Have I missed a reply on why this belongs here instead?